neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/remote/http/impl/HttpNeo4jClient.java

```diff
@@ -25,7 +30,11 @@
  *
  * @author Davide D'Alto
  */
-public class RemoteNeo4jClient implements AutoCloseable {
+public class HttpNeo4jClient {
```

Comment:
Any particular reason why you removed the AutoCloseable interface? Just curious.